### PR TITLE
Return 409 if there is a conflict when activating

### PR DESCRIPTION
**Pull request by hmusum**
Today, you get 400 Bad Request when activation fails due to a conflict. Using 409 seems correct and will also gives us the possibility to retry deployment if we want to avoid deployment failures in this scenario.

Doc will be updated in another PR.

### CODE REVIEW

**Comment by hmusum** [comment]
ping @bratseth

File: configserver/src/main/java/com/yahoo/vespa/config/server/ActivationException.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+package com.yahoo.vespa.config.server;
+
+/**
+ * Exception used when activation cannot be done because activation is for
+ * an older session than the one that is active now or because current active
+ * session has changed since the session to be activated was created
+ *
+ * @author hmusum
+ */
+public class ActivationException extends RuntimeException {
```
Comment (bratseth) [inline configserver/src/main/java/com/yahoo/vespa/config/server/ActivationException.java]: Too general? ActivationConflictException?

Comment (hmusum) [reply configserver/src/main/java/com/yahoo/vespa/config/server/ActivationException.java]: Yeah, good point, will change it

**Comment by bratseth** [comment]
👍

**Comment by gjoranv** [comment]
I suspect that this broke the deploy_v2 system test. Please note that the 'git (last systest)' link on factory not necessarily can be trusted, as it seems to include commits that were merged before the last successful run of the deploy_v2 test.

**Comment by hmusum** [comment]
Thanks!

**Pull request merged**